tensorflow/core/util/gpu_launch_config.h

```diff
@@ -142,10 +142,12 @@ inline GpuLaunchConfig GetGpuLaunchConfig(int work_element_count,
   config.block_count = block_count;
   return config;
 }
+#ifndef TENSORFLOW_USE_ROCM
 inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
```

Comment:
Already defined in line 198, we should remove it

Reply:
same reasoning+comment  as for the `GetGpu2DLaunchConfig`